src/sql/base/browser/ui/editableDropdown/dropdown.ts

```diff
@@ -129,6 +129,10 @@ export class Dropdown extends Disposable {
 			ariaLabel: this._options.ariaLabel
 		});
 
+		// Clear title from input box element (defaults to placeholder value) since we don't want a tooltip for the selected value
+		// in the text box - we already have tooltips for each item in the dropdown itself.
+		this._input.inputElement.title = '';
+
```

Comment:
Whats the reason for not wanting the title to be the placeholder? thats standard across vscode since the placeholder should be a description of the input.

Reply:
The issue is how we're using it for database dropdowns. The placeholder value in that case is <default> - which doesn't make sense as a tooltip (especially after they've chosen an actual database)

We could have the title be set to some different value for that case though for accessibility - "Select a database" or something.

Reply:
We correctly set the aria-label to "Database" so this still works correctly with the screen reader. We just no longer have the tooltip with "<default>" which is what I was originally going for.

Reply:
It makes sense if the placeholder is actually an unselectable placeholder that describes the drop down but doesn't work for drop downs where there's a default selection and not a placeholder. Maybe there's some way to differentiate them? But unless this is breaking any existing dropdowns where we rely on that functionality the change seems good to me